File: app/src/main/java/net/radityalabs/contactapp/presentation/util/WidgetUtil.java
package net.radityalabs.contactapp.presentation.util;

import android.content.Context;

/**
 * Created by radityagumay on 4/15/17.
 */

public class WidgetUtil {

    private static float density(Context context) {
        return context.getResources().getDisplayMetrics().density;
    }

    public static float pxToDp(Context context, int pixel) {
        return pixel / density(context);
    }

    public static float dpToPx(Context context, int dp) {
        return dp * density(context);
    }
}

File: app/src/main/java/net/radityalabs/contactapp/presentation/util/WidgetUtil.kt
package net.radityalabs.contactapp.presentation.util

import android.content.Context

/**
 * Created by radityagumay on 4/15/17.
 */

object WidgetUtil {

    private fun density(context: Context): Float {
        return context.resources.displayMetrics.density
    }

    fun pxToDp(context: Context, pixel: Int): Float {
        return pixel / density(context)
    }

    fun dpToPx(context: Context, dp: Int): Float {
        return dp * density(context)
    }
}
